File: rasterizer-dotnet-vb/MultiPageTiff.vb
Imports ceTe.DynamicPDF.Rasterizer
Class MultiPageTiff

    Friend Shared Sub Run()
        Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
        Dim fixedImageSize As New FixedImageSize(595, 841)
        rasterizer.DrawToMultiPageTiff(Program.GetOutputDocPath("output-multipage-tiff.tif"), ImageFormat.TiffWithLzw, fixedImageSize)
    End Sub

End Class

File: rasterizer-dotnet-examples-csharp/MultiPageTiff.cs
using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    class MultiPageTiff
    {
        internal static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            FixedImageSize fixedImageSize = new FixedImageSize(595, 841);
            rasterizer.DrawToMultiPageTiff(Program.GetOutputDocPath("output-multipage-tiff.tif"), ImageFormat.TiffWithLzw, fixedImageSize);
        }
    }
}